src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.cs

```diff
@@ -32,6 +32,16 @@ protected override async Task<InsertionPoint> GetInsertionPointAsync(SemanticDoc
             var root = await document.Document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
             var basePosition = root.FindToken(position);
 
+            // Check if we are extracting a local function and are within a local function
+            if (ExtractLocalFunction)
+            {
+                var localMethodNode = basePosition.GetAncestor<LocalFunctionStatementSyntax>();
+                if (localMethodNode is object)
+                {
+                    return await InsertionPoint.CreateAsync(document, localMethodNode, cancellationToken).ConfigureAwait(false);
```

Comment:
this comment isn't really helpful.  if we're extracting an local function and we're in a local function... then ... what?  why is that case relevant/important?  what are we doing special here?